backend/typescript/services/implementations/campService.ts

```diff
@@ -101,6 +101,47 @@ class CampService implements ICampService {
     }
   }
 
+  async getCampById(campId: string): Promise<CampDTO> {
+    let camp: Camp | null;
+
+    try {
+      camp = await MgCamp.findById(campId);
+
+      if (!camp) {
+        throw new Error(`Camp' with campId ${campId} not found.`);
+      }
+    } catch (error: unknown) {
+      Logger.error(`Failed to update camp. Reason = ${getErrorMessage(error)}`);
```

Comment:
update error message!
```suggestion
      Logger.error(`Failed to get camp. Reason = ${getErrorMessage(error)}`);
```